definitions/views/atomicmarket_template_prices_master.sql

```diff
@@ -1,15 +0,0 @@
-CREATE OR REPLACE VIEW atomicmarket_template_prices_master AS
```

Comment:
Do you want to keep the views as they are, or should I change them to select from the new static table?

Reply:
Ideally it should be changed to the static table but I think we need to keep the other view because otherwise the initial setup on an empty database will fail